sci-biology/discrover/discrover-1.5.2.ebuild

```diff
@@ -37,16 +37,19 @@ DEPEND="${RDEPEND}
 		dev-texlive/texlive-latexrecommended
 		media-gfx/imagemagick
 	)
+	lto? ( >=sys-devel/gcc-4.8 )
 "
 src_configure() {
 	local mycmakeargs=(
 		$(cmake-utils_use_with cairo CAIRO)
 		$(cmake-utils_use_with dreme DREME)
 		$(cmake-utils_use_with doc DOC)
+		$(cmake-utils_use_with lto LTO)
```

Comment:
That seems to be a good idea! I think I might do that.

My initial thought was that DEPENDing on >=sys-devel/gcc-4.8.0 conditionally on the USE flag lto (as is done in the ebuild now) might ensure usage of gcc >= 4.8. But I realized now that having emerged gcc >= 4.8 doesn't necessarily mean that it's configured to be used. Hence I'm currently emerging gcc < 4.8 to test that assumption. Anyway, I believe your suggestion is the solution.